account/src/main/java/org/killbill/billing/account/api/user/DefaultAccountUserApi.java

```diff
@@ -232,4 +234,9 @@ public Account apply(final AccountModelDao input) {
                                                                              }
                                                                          }));
     }
+
+    @Override
+    public List<AuditLogWithHistory> getAuditLogsWithHistoryForId(final UUID accountId, final AuditLevel auditLevel, final TenantContext tenantContext) throws AccountApiException {
+        return accountDao.getAuditLogsWithHistoryForId(accountId, auditLevel, internalCallContextFactory.createInternalTenantContextWithoutAccountRecordId(tenantContext));
```

Comment:
Use `createInternalTenantContext(final UUID accountId, final TenantContext context)` instead of `createInternalTenantContextWithoutAccountRecordId`.

See also comment here: https://github.com/killbill/killbill/blob/5d63b9b5e88f2c603046b9f60139a93caee7fbbd/util/src/main/java/org/killbill/billing/util/callcontext/InternalCallContextFactory.java#L106 ;-)